tooling/nargo_cli/tests/execution_success/workspace_reexport_bug/library2/src/lib.nr

```diff
@@ -0,0 +1,5 @@
+// When we re-export this type from another library and then use it in
+// main, we get a panic
+struct ReExportMeFromAnotherLib {
+    x : Field,
+}
```

Comment:
This is the struct that we will reexport in `library`, and then use that in `binary`. The workaround here would be to have binary import `library2` directly.

However the issue also pops up if you want to define a new struct in `library` that uses `ReExportMeFromAnotherLib` as one of its fields. This is just a simplified example.